lib/app_networking.dart

```diff
@@ -60,7 +58,7 @@ class NetworkHelper {
       title: const Text(LoginConstants.silentLoginFailedTitle),
       content: Text(LoginConstants.silentLoginFailedDesc),
       actions: [
-        FlatButton(
+        TextButton(
```

Comment:
@kwgong Please update the TextButtons with this style to resolve the problem reported on the issue [here](https://github.com/UCSD/campus-mobile/issues/1457#issuecomment-872577424):
```
style: TextButton.styleFrom(
        primary: Theme.of(context).buttonColor,
      ),
```